.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "modules/taskcluster-terraform"]
-	path = modules/taskcluster-terraform
-	url = https://github.com/taskcluster/taskcluster-terraform
+[submodule "taskcluster"]
+	path = modules
+	url = git@github.com:taskcluster/taskcluster.git
```

Comment:
This URL format requires SSH.  We probably all have SSH set up, though, so maybe not a big deal..